.github/ISSUE_TEMPLATE/support-create-a-new-fork.md

```diff
@@ -0,0 +1,17 @@
+---
+name: 'Support: Create a new Fork'
```

Comment:
Suggestion for clarification:
- Create a fork in the consortium GitHub organization

Reply:
adapted your proposals to make it clear where we create things, see https://github.com/eclipse-tractusx/sig-infra/pull/10/commits/54e87d6f9a8ee097475f50af445802f520002e39